### ISSUE TITLE
write vignette

### ISSUE THREAD

**Issue by pearsonca** [pearsonca/hbmPRNG#4]

**Comment by pearsonca** [comment]
written, works, shows effects (-ish? bunch of non-epidemics could potentially be pruned - is that fair play?)

 - [ ] slow - reasonable to do pure binomial simulation version of matching? that's a plausible non-cheat in compartmental land, but i think neglects real individual uncertainty. that's perhaps a useful element to highlight?
 - [ ] probably going to choke on CRAN process, since using packages in the vignette build that aren't required for package build. legal for those to just go in suggests?

**Comment by seabbs** [comment]
yup legal to put packages used in vignettes into suggests and can pre-compile the vignettes to avoid slow run times

**Comment by pearsonca** [comment]
actual yielded data is pretty thin - how about
 - `eval = FALSE` on the slow blocks,
 - do the same long-run steps in data-raw,
 - store the data in inst? seems silly to add any namespace items by putting it in actual data
 - have the vignette load data from the package files instead

it's a bit clunky to change out the simulation / data part of the vignette when that needs updating, but that seems likely to be firm enough soon
